### PR TITLE
JS Build Improvements

### PR DESCRIPTION
**Pull request by davmillar**
Mainly updating of assets and compilation into a single JS file.

### CODE REVIEW

File: scripts/b64.js [hunk 1]
```diff
@@ -1,127 +0,0 @@
...
- * Fox, nobbler, stensi, Matteo, Riddler (http://www.frontierwebdev.com/),
- * Tomasz Wesolowski, T.J. Leahy, rezna, Eric Nagel, Alexander M Beedie, baris
- * ozdil, Greg Frazier, Bobby Drake, Ryan W Tenney (http://ryan.10e.us), Tod
- * Gentille, Rafał Kukawski, FremyCompany, Manish, Cord, fearphage
- * (http://http/my.opera.com/fearphage/), Victor, Brant Messenger
- * (http://www.brantmessenger.com/), Matt Bradley, Luis Salazar
- * (http://www.freaky-media.com/), Tim de Koning, taith, Rick Waldron, Mick@el
- * 
- * Dual licensed under the MIT (MIT-LICENSE.txt)
- * and GPL (GPL-LICENSE.txt) licenses.
```
Comment (morgajel) [inline scripts/b64.js]: Uh don't you need to keep this in the file?

Comment (davmillar) [inline scripts/b64.js]: @morgajel the whole file is being removed and replaced with two separate files for the needed functions. GitHub doesn't make this terribly obvious in diffs though.

Comment (morgajel) [inline scripts/b64.js]: ah, cool, as long as its in here somewhere.

Comment (davmillar) [inline scripts/b64.js]: Yeah, the new files each have their commentary, credits, and licensing intact. They're the base64_encode and utf8_encode files down lower in the diff.

**Pull request merged**